### PR TITLE
Refactored RefactorKata

### PR DESCRIPTION
**Pull request by HunterAW**
Refactored RefactorKata to be reviewed.

### CODE REVIEW

File: RefactorKata/Program.cs [hunk 1]
```diff
@@ -1,41 +1,36 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Data.SqlClient;
 
 namespace RefactorKata
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main()
```
Comment (codytwinton) [inline RefactorKata/Program.cs]: I advise against removing the params from `Main` as they can be used by the project options. Please re-add `string[] args`.

File: RefactorKata/Program.cs [hunk 2]
```diff
@@ -1,41 +1,36 @@
...
+            using (var conn = new SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;"))
             {
-                Console.WriteLine(products[i].name);
+                var cmd = conn.CreateCommand();
+                cmd.CommandText = "select * from Products";
+
+                var reader = cmd.ExecuteReader();
+                var products = new List<Product>();
+
+                //TODO: Replace with Dapper once Dapper is tought
```
Comment (codytwinton) [inline RefactorKata/Program.cs]: Misspelled taught. :D

File: RefactorKata/Program.cs [hunk 3]
```diff
@@ -1,41 +1,36 @@
...
+
             }
         }
     }
-    public class Product
-    {
-        public string name;
-        public string Name { get { return name; } set { name = value; } }
-    }
+
```
Comment (codytwinton) [inline RefactorKata/Program.cs]: Remove this whitespace

**Comment by HunterAW** [comment]
Notes on the last commit addressed.

**Pull request closed without merging**